python/Gaffer/Application.py

```diff
@@ -86,6 +93,10 @@ def root( self ) :
 	## Called to run the application and return a status value.
 	def run( self ) :
 
+		if self.parameters()["help"].getValue() :
+			os.system( "gaffer -help " + self.path )
```

Comment:
This is iffy. The Application class isn't necessarily called by the bin/gaffer.py wrapper anyway (we sometimes launch applications direct from Python, like we do in Caribou), and even when it is, launching a separate process just to print a help string is an unnecessary overhead (and confusing for users if that leads to duplicate warnings from config files etc).

Maybe this could be achieved more cleanly by using the printing code from bin/gaffer.py directly here, and then implementing `gaffer -help appName` by setting the help parameter and running the app? Or maybe we don't really need two ways of doing the same thing?